app/component/map/city-bike-marker.cjsx

```diff
@@ -0,0 +1,73 @@
+React         = require 'react'
+isBrowser     = window?
+Marker        = if isBrowser then require 'react-leaflet/lib/Marker'
+Popup         = if isBrowser then require './dynamic-popup'
+#Popup        = if isBrowser then require 'react-leaflet/lib/Popup'
+L             = if isBrowser then require 'leaflet'
+CityBikePopup = require './city-bike-popup'
+provideContext = require 'fluxible-addons-react/provideContext'
+intl          = require 'react-intl'
+Icon          = require '../icon/icon'
+
+
+STOPS_SMALL_MAX_ZOOM = 15
+
+# Small icon for zoom levels <= 15
+smallIconSvg = """<svg viewBox="0 0 8 8">
+    <circle class="stop-small" cx="4" cy="4" r="3" stroke-width="1"/>
+  </svg>"""
+
+class CityBikeMarker extends React.Component
```

Comment:
This seems like an awful lot of duplicated code, should we have a more generic marker class, which we could just supply icons and behaviour?